index.js

```diff
@@ -2,13 +2,21 @@
 
 const fp = require('fastify-plugin')
 const { createHash } = require('crypto')
+const { xxHash32 } = require('./xxhash')
 const fnv1a = require('./fnv1a')
 
 function buildHashFn (algorithm = 'fnv1a') {
   if (algorithm === 'fnv1a') {
     return (payload) => '"' + fnv1a(payload).toString(36) + '"'
   }
 
+  if (algorithm === 'xxhash') {
+    return (payload) => {
+      const bufferPayload = payload instanceof Buffer ? payload : Buffer.from(payload, 'utf8')
+      return '"' + xxHash32(bufferPayload, 0).toString(16) + '"'
```

Comment:
it seems that this code could be updated to this one right now, removing the elvis operator:
```suggestion
      return '"' + xxHash32(payload, 0).toString(16) + '"'
```

Reply:
You are absolutely right! I forgot to change that and this will have performance implications. Fixing this and re-running the benchmarks later today!

Thanks a lot for spotting this!